aggregator/.env.example

```diff
@@ -40,3 +40,7 @@ PRIORITY_FEE_PER_GAS=0
 PREVIOUS_BASE_FEE_PERCENT_INCREASE=2
 
 BUNDLE_CHECKING_CONCURRENCY=8
+
+IS_OPTIMISM=false
+OPTIMISM_GAS_PRICE_ORACLE_ADDRESS=0x420000000000000000000000000000000000000F
+OPTIMISM_L1_BASE_FEE_PERCENT_INCREASE=2
```

Comment:
@voltrevo I used this default value based on `PREVIOUS_BASE_FEE_PERCENT_INCREASE` per docstring, let me know if we should use something else.